docs/topics/Gateway.md

```diff
@@ -781,6 +790,29 @@ A user's presence is their current state on a guild. This event is sent when a u
 >info
 >The streaming type currently only supports Twitch. Only `https://twitch.tv/` urls will work.
 
+###### Game Timestamps
+
+| Field | Type | Description |
+|-------|------|-------------|
+| start? | int | Epoch millis of start |
```

Comment:
`unix time (in milliseconds) of when the activity started` would be a lot clearer and more in-line with our description of `since` in status updates